conf/eddie.config

```diff
@@ -58,3 +45,4 @@ singularity {
 }
 
 includeConfig 'singularity.config'
```

Comment:
Personally think that container.config should be separate to singularity.config, and you can have:

includeConfig 'container.config' into eddie.config and remove it from the 'singularity.config' specification. This is something I will update for ultra2 use when I implement TarGene.

Reply:
Is there any case where you would use singularity without containers?

Reply:
you might just need to specify them in a different way. for example - with all of us, we will route through docker hub a different way to how it is currently done in the container.config file. And on ultra2, we need to point at images deposited on ultra2 in order to run, as ultra2 has no access to the internet. See https://github.com/TARGENE/ld-block-removal/blob/main/conf/local_container.config for an example.

Reply:
However, if you don't see that useful now, I can always add that on my own branch in the future when I update TarGene for these use cases.

Reply:
I've split out eddie.config like this, https://github.com/TARGENE/ld-block-removal/blob/main/conf/eddie.config but retaining the same structure you have required in targene-pipeline.

Reply:
Sounds good, I've made the change.